app/views/layouts/_flashes.html.erb

```diff
@@ -0,0 +1,7 @@
+<% unless flash.empty? %>
+  <div class="flashes">
+    <% flash.each do |type, message| %>
+      <div class="<%= flash_htmlclass_for type %>"><%= message.to_s %></div>
```

Comment:
I considered that the flash messages can be only `:success` or `:error` as we agreed in the scrum.